### PR TITLE
Add environment variables to upload codecov results without a token

### PR DESCRIPTION
**Pull request by jbeezley**
This worked in https://app.circleci.com/pipelines/github/girder/girder/1036/workflows/de720239-3ce3-4d19-a16b-dee390870e26/jobs/28677.

### CODE REVIEW

**Comment by zachmullen** [comment]
Looks like this doesn't work for codecov

**Comment by zachmullen** [comment]
Actually coverage uploaded just fine... no idea why the codecov status checks are failing

**Review by zachmullen — APPROVED** [review]

File: tox.ini [hunk 1]
```diff
@@ -73,14 +73,15 @@ commands =
...
+passenv =
+    CI
+    CIRCLE*
 deps =
     codecov
 commands =
     codecov \
         --disable search pycov gcov \
         --root girder \
-        --required \
```
Comment (zachmullen) [inline tox.ini:82]: I think we should keep `--required`, right? Otherwise it fails silently

**Review by zachmullen — CHANGES_REQUESTED** [review]

File: tox.ini (hunk 1 above)
Comment (jbeezley) [reply tox.ini:82]: Right.  Added back.

**Review by zachmullen — APPROVED** [review]

**Pull request merged**